index.js

```diff
@@ -68,7 +68,21 @@ module.exports = function (str) {
   } else if ((match = str.match(new RegExp(regexps['re1'], 'i')))) {
     let days = match[1];
     let period = match[2];
-
+    if (period === 'year') {
+      let year = new Date().getFullYear() - days;
+      if (year < 0) {
+        return 'Before B.C.';
```

Comment:
The return value should always be a `[<boolean>, <date-obj or null>]`

Reply:
For B.C what should I return: [true, null] or [false, null] or year in minus?

Reply:
I will take care of it.

Reply:
Thank You